init/bash

```diff
@@ -13,7 +13,7 @@ source $(dirname "$BASH_SOURCE")/eessi_environment_variables
 # only continue if setting EESSI environment variables worked fine
 if [ $? -eq 0 ]; then
 
-    export PS1="[EESSI $EESSI_VERSION] $ "
+    export PS1="[EESSI $EESSI_VERSION]$PS1"
```

Comment:
Maybe `{EESSI 2023.06} ` would even be better (with a space in between) since `[...]` is already quite common in `$PS1`, so it stands a bit out more?

Compare:

```
[EESSI 2023.06][escobar@sei68 ~]$
```

with

```
{EESSI 2023.06} [escobar@sei68 ~]$
```

Reply:
I have no strong opinion. Whatever that doesn't override my PS1 works for me.

I have no computer access now. I will update the PR with your suggestion later today.